src/main/target/TINYFISH/target.h

```diff
@@ -66,7 +66,7 @@
 #define UART3_TX_PIN            PB10
 #define UART3_RX_PIN            PB11
 
-#define SBUS_TELEMETRY_UART     SERIAL_PORT_USART2
+#define SBUS_AND_TELEMETRY_UART SERIAL_PORT_USART2
```

Comment:
See https://github.com/cleanflight/cleanflight/blob/master/src/main/io/serial.c#L133-L138 and https://github.com/cleanflight/cleanflight/search?utf8=%E2%9C%93&q=SERIALRX_UART&type=